src/components/Categories/Categories.jsx

```diff
@@ -1,9 +1,15 @@
+import { useDispatch } from 'react-redux';
 import './Categories.css';
+import { statusAction } from '../../redux/categories/categories';
 
-const Categories = () => (
-  <div className="categories">
-    <button type="button">Check status</button>
-  </div>
-);
+const Categories = () => {
+  const dispatch = useDispatch();
+
+  return (
+    <div className="categories">
+      <button type="button" onClick={() => dispatch(statusAction())}>Check status</button>
```

Comment:
- [ ] Clicking on this button should dispatch an action that would show the text "under construction" 💯 Kindly make the necessary changes to satisfy this requirement 👇 

![Animation](https://user-images.githubusercontent.com/50721479/207429191-cd377ded-1dc4-4f0e-a567-f8bf66ee4017.gif)